README.md

```diff
@@ -158,16 +164,16 @@ To use the setup parameter `minimum_confidence`, you have multiple options:
 
 For example:
 ```js
-controller.hears(['.*'], ['direct_message', 'direct_mention', 'mention', 'message_received'], function(bot, message) {
+controller.hears(['.*'], ['direct_message', 'direct_mention', 'mention', 'message_received'], async function(bot, message) {
   if (message.watsonError) {
-    bot.reply(message, "Sorry, there are technical problems.");     // deal with watson error
+    await bot.reply(message, "Sorry, there are technical problems.");     // deal with watson error
   } else {
     if (message.watsonData.intents.length == 0) {
-      bot.reply(message, "Sorry, I could not understand the message.");     // was any intent recognized?
+      await bot.reply(message, "Sorry, I could not understand the message.");     // was any intent recognized?
     } else if (message.watsonData.intents[0].confidence < watsonMiddleware.minimum_confidence) {
-      bot.reply(message, "Sorry, I am not sure what you have said.");      // is the confidence high enough?
+      await bot.reply(message, "Sorry, I am not sure what you have said.");      // is the confidence high enough?
```

Comment:
Why you add await ? this is not promise ..

Reply:
it is now - https://github.com/howdyai/botkit/blob/ead1d91993f4a67375449ee9db6fe3f74e556ceb/packages/botkit/src/botworker.ts#L130

Reply:
Sorry ! i have not say the new type ...